src/types/markers.js

```diff
@@ -9,17 +9,52 @@ import type { IndexIntoStackTable, IndexIntoStringTable } from './profile';
 
 // Provide different formatting options for strings.
 export type MarkerFormatType =
-  | 'string' // "Label: Some String"
-  | 'bytes' // "Label: 5mb"
+  // ----------------------------------------------------
+  // String types.
+
+  // Show the URL, and handle PII sanitization.
+  | 'url'
+  // Show the file path, and handle PII sanitization.
+  | 'file-path'
+  // Important, do not put URL or file path information here, as it will not be
+  // sanitized. Please be careful with including other types of PII here as well.
+  // e.g. "Label: Some String"
+  | 'string'
+
+  // ----------------------------------------------------
+  // Numeric types
+
+  // For time data that represents a duration of time.
+  // e.g. "Label: 5s, 5ms, 5μs"
+  | 'duration'
```

Comment:
Are we storing as a millisecond, or something else in the profile? Would be good to add this to the comment too.